dev-embedded/libftdi/libftdi-1.5.recipe

```diff
@@ -5,36 +5,40 @@ library is linked with your program in userspace, no kernel driver required.\
 libFTDI works perfectly with Linux, Windows, MacOS X and BSD variants thanks to \
 libusb."
 HOMEPAGE="https://www.intra2net.com/en/developer/libftdi/index.php"
-COPYRIGHT="2003-2017 Intra2net AG and the libftdi developers"
+COPYRIGHT="2003-2020 Intra2net AG and the libftdi developers"
 LICENSE="GNU LGPL v2.1
 	GNU GPL v2"
-REVISION="8"
+REVISION="1"
 SOURCE_URI="https://www.intra2net.com/en/developer/libftdi/download/libftdi1-$portVersion.tar.bz2"
-CHECKSUM_SHA256="ec36fb49080f834690c24008328a5ef42d3cf584ef4060f3a35aa4681cb31b74"
+CHECKSUM_SHA256="7c7091e9c86196148bd41177b4590dccb1510bfe6cea5bf7407ff194482eb049"
 SOURCE_DIR="libftdi1-$portVersion"
 PATCHES="libftdi-$portVersion.patchset"
 
 ARCHITECTURES="!x86_gcc2 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 
+libVersion="2.5.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
```

Comment:
@korli compat string for this is fine for libftdi1 but libftddipp1 provides a so link with compat = 3 (libftdipp1.so.3 -> libftdipp1.so.2.5.0), should there be some change for this?

Reply:
@Begasus it's OK so. Alternatively one can use "3.0.0 >= 3". It's only important if some other package uses this library..

Reply:
Thanks @korli , I thought it was OK also, but wanted to double check ;) :+1: